kie-osgi/kie-karaf-itests/src/test/java/org/kie/karaf/itest/blueprint/KieBlueprintScannerReimportIntegrationTest.java

```diff
@@ -80,6 +83,16 @@ private static void setup() {
         kieScannerTestUtils.tearDown();
     }
 
+    @BeforeClass
+    public static void generalSetup() {
+        System.setProperty(KieServerConstants.KIE_SERVER_MODE, "regular");
```

Comment:
```suggestion
        System.setProperty(KieServerConstants.KIE_SERVER_MODE, KieServerMode.REGULAR.name());
```

Reply:
@tomasdavidorg I already tried that but it was causing a ClassNotFoundException error during the execution, weird but with this change the test works